### DISCUSSION TITLE
blog/2023-02-06-causal-inference-with-potential-outcomes-bootcamp/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#6] [Comments]
# blog/2023-02-06-causal-inference-with-potential-outcomes-bootcamp/

Part 2 of the GLM and causal inference series.

https://timely-flan-2986f4.netlify.app/blog/2023-02-06-causal-inference-with-potential-outcomes-bootcamp/

**Comment by sjwild** [comment]
Another good post. I don't have a lot of substantive comments. A few minor ones:
* The beta coefficients __can be__ unbiased estimates of the population-level causal effects. There are situations they might not be, such as there is selection bias. This might be worth pointing out, even if you hand wave it away.
* I like that you said ATE == PATE in this post. I like it when the ATE is clearly identified.
* I love the meme.
* I would edit this sentence to use _estimate_ instead of _estimand_: "...so you can properly express the uncertainty of your estimand with 95% intervals..."
* _Estimate_ is spelt wrong in the following sentence: "Now we have a nice standard error and 95% interval range for the estiamte of our estimand..."
* You spend several paragraphs looking at `ols1` and computing $\mathbb{E}({y_i}^1)$ and $\mathbb{E}({y_i}^0)$ for all 42 subjects. The value is the same for each subject for each $\mathbb{E}({y_i}^1)$ and $\mathbb{E}({y_i}^0)$, so it might be worth a sentence or two to explain why. It makes sense once you show the results from `ols2`, but before then some readers may be confused.

**Reply by ASKurz** [reply]
Anticipating the next post, I'm starting to think it might make sense to show the $\hat y_1^1$ and $\hat y_1^0$ estimates and their contrasts in a couple plots.

**Reply by ASKurz** [reply]
Okay, my edits based on your feedback, @sjwild, are up.

**Comment by sjwild** [comment]
I like the images. The use of rank plots was good, and I do think it adds something.

I like this:
* "Thus, the average causal effect of the experimental condition relative to the waitlist control is a reduction of about 2 and a half pounds, with with a very wide 95% confidence interval range spanning from a reduction of 13 pounds to an increase of 8 pounds"

One thing I go back and forth about:
* I'm not sure whether to identify ${\beta_1}$ as the estimate, with OLS being the estimator. This could just me being overly pedantic, or it could be me probably being wrong.

**Reply by ASKurz** [reply]
If we're being pedantic, I'd say the point estimate value for $\beta_1$ is the estimate and the entire linear model via OLS is the estimator, which would make the parameter $\beta_1$ part of the estimator.

**Reply by sjwild** [reply]
Fair enough. I told you I was probably wrong :-p

**Comment by chelseaparlett** [comment]
Love the color coding of formulas, would love to see more of that. No other comments, this is lovely!
